config/unix-gfortran.cmake

```diff
@@ -1,24 +1,20 @@
-#-----------------------------*-cmake-*----------------------------------------#
+#-----------------------------*-cmake-*-----------------------------------------------------------#
 # file   config/unix-gfortran.cmake
 # author Kelly Thompson
 # date   2010 Sep 27
 # brief  Establish flags for Unix/Linux - Gnu Fortran
-# note   Copyright (C) 2016-2020 Triad National Security, LLC.
-#        All rights reserved.
-#------------------------------------------------------------------------------#
+# note   Copyright (C) 2016-2020 Triad National Security, LLC. All rights reserved.
+#-------------------------------------------------------------------------------------------------#
 
 include_guard(GLOBAL)
 
 if( NOT Fortran_FLAGS_INITIALIZED )
-  # gfortran < 4.3 won't compile Draco
   # gfortran < 4.7 won't compile Jayenne
-  set( CMAKE_Fortran_COMPILER_VERSION ${CMAKE_Fortran_COMPILER_VERSION} CACHE
-    STRING "Fortran compiler version string" FORCE )
-  if( "${CMAKE_Fortran_COMPILER_VERSION}" STRLESS "4.7" )
```

Comment:
`STRLESS` is s string comparison.  gfortran-10's version starts with `1 < 4`.  Thus failing in Appveyor CI.